src/main/java/seedu/address/logic/parser/ArgumentMultimap.java

```diff
@@ -84,11 +84,20 @@ public boolean contains(Prefix prefix) {
     }
 
     /**
-     * Checks if any number of prefixes is present in the ArgumentMultimap object
+     * Checks if any number of the prefixes specified is present in the ArgumentMultimap object
      * @param prefixes to check
      * @return true if prefix/s present and false otherwise
      */
-    public boolean isPrefixPresent(Prefix... prefixes) {
-        return Stream.of(prefixes).allMatch(prefix -> contains(prefix));
+    public boolean isAnyPrefixPresent(Prefix... prefixes) {
+        return Stream.of(prefixes).anyMatch(prefix -> contains(prefix));
+    }
+
+    /**
+     * Checks if all prefixes specified is present in the ArgumentMultimap object
+     * @param prefixes to check
+     * @return true if prefix/s present and false otherwise
+     */
+    public boolean isAllPrefixesPresent(Prefix... prefixes) {
```

Comment:
Is this method still in use?

Reply:
its not duplicate, it checks if all prefixes are in argMultimap (unlike the other one which checks if any of the prefixes is in argMultimap), will be useful for some bug fixes im working on